Changelog.md

```diff
@@ -38,6 +38,7 @@ __Bugfixes__
 - Adds more decimal places in output files as needed for simulations with shorter timesteps and/or abbreviated run periods.
 - Timeseries output fixes: some outputs off by 1 hour; possible negative combi boiler values.
 - Fixes range hood ventilation interaction with infiltration to take into account the location of the cooking range.
+- Fixes incorrect outside boundary condition for shared gable walls of cathedral ceilings, now set to adiabatic.
```

Comment:
If this fixed a bug, why isn't there a change to the gable walls defined in the base-bldgtype-single-family-attached.xml sample file?

Reply:
That file does not have cathedral ceilings

Reply:
Right. Good point. Do you mind creating a new base-bldgtype-single-family-attached-atticroof-cathedral.xml based on base-bldgtype-single-family-attached.xml? Seems like a good test file to have.